log2ram

```diff
@@ -52,16 +52,16 @@ wait_for () {
 
 createZramLogDrive () {
 	# Check Zram Class created
-	if [ ! -d "/sys/class/zram-control" ]; then
-		modprobe zram
+	if modprobe --first-time --verbose zram; then
 		RAM_DEV='0'
 	else
 		RAM_DEV=$(cat /sys/class/zram-control/hot_add)
 	fi
 	echo ${COMP_ALG} > /sys/block/zram${RAM_DEV}/comp_algorithm
 	echo ${LOG_DISK_SIZE} > /sys/block/zram${RAM_DEV}/disksize
 	echo ${SIZE} > /sys/block/zram${RAM_DEV}/mem_limit
-	mke2fs -t ext4 /dev/zram${RAM_DEV}
+	mke2fs -v -t ext4 -O ^has_journal -s 1024 -L log2ram$RAM_DEV /dev/zram${RAM_DEV}
```

Comment:
Is that a typo error ?

Reply:
Yeah I just put it there for a laugh

Reply:
can you remove the '^' ?

Reply:
No as that means no journal ^ that is the command for no journal,  has_journal is the oppisite!

Reply:
My bad.